projects/dsp-ui/README.md

```diff
@@ -1,26 +1,288 @@
-# @knora/ui
+# @dasch-swiss/dsp-ui
 
-Reusable Angular elements for Angular applications based on knora-api.
+The modules help to create a graphical user interface, a web application to use [DSP-API](https://api.knora.org) in a quick and simple way. The modules are written in Typescript to use them with **[Angular](https://angular.io) (version 9)**. We decided to style components and directives with [Angular Material design](https://material.angular.io).
```

Comment:
This is redundant. I think you should link to this README from the app's README.

Reply:
In this case we'll have a link to another README on npm instead of a short description. This could be confusing...

Reply:
Then simply the other way round? So from the app's README to the lib's README.